<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Controller</class>
 <widget class="QWidget" name="Controller">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>509</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Controller</string>
  </property>
  <widget class="QTextEdit" name="te_send">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>60</y>
     <width>241</width>
     <height>171</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>接收内容：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>40</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>发送内容：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_listen">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>监听</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lE_port">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>10</y>
     <width>41</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="te_recv">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>231</width>
     <height>201</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>10</y>
     <width>31</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>端口：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_send">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>240</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>发送</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
